sqlitecluster/SQLiteNode.cpp

```diff
@@ -1277,13 +1277,22 @@ void SQLiteNode::_onMESSAGE(SQLitePeer* peer, const SData& message) {
             SINFO("Received PING from peer '" << peer->name << "'. Sending PONG.");
             SData pong("PONG");
             pong["Timestamp"] = message["Timestamp"];
-            peer->sendMessage(pong.serialize());
+            peer->sendMessage(pong);
             return;
         } else if (SIEquals(message.methodLine, "PONG")) {
             // Latency must be > 0 because we treat 0 as "not connected".
             peer->latency = max(STimeNow() - message.calc64("Timestamp"), 1ul);
             SINFO("Received PONG from peer '" << peer->name << "' (" << peer->latency/1000 << "ms latency)");
             return;
+        } else if (SIEquals(message.methodLine, "NODE_LOGIN")) {
+            // Do nothing, this keeps this code from warning until NODE_LOGIN is deprecated.
```

Comment:
Would it be useful to log something here?

Reply:
No, I expect this to happen because we added this line: https://github.com/Expensify/Bedrock/pull/1999/files#diff-2d1c5f24a4b7ebc3076630d8ee3a1155ad5e7789abe72cff977e615541c319d6R2627

I want this to get called when this message is *LOGIN* but it doesn't need to do anything when it's *NODE_LOGIN*, it can just be quiet until we take that out.

Reply:
I like the idea of at least logging that we are ignoring it